input/fsh/SD_CodedContentBundles.fsh

```diff
@@ -51,3 +51,20 @@ Description: "Demographic Coded Content Bundle (Bundle): A bundle containing ins
 * entry ^slicing.description = "Slicing based on the profile"
 * insert BundleSlice(  CodedRaceAndEthnicity,  0, 1,  CodedRaceAndEthnicity,  CodedRaceAndEthnicity,  CodedRaceAndEthnicity)
 * insert BundleSlice(  InputRaceAndEthnicity,  0, 1,  InputRaceAndEthnicity,  InputRaceAndEthnicity,  InputRaceAndEthnicity)
+
+Profile: IndustryOccupationCodedContentBundle
+Parent: Bundle
+Id: vrdr-industry-occupation-coded-bundle
+Title: "Industry Occupation Coded Content Bundle"
+Description: "Industry and Occupation Coded Content Bundle (Bundle):  A bundle containing  coded industry and occupation data for a decedent.   This bundle is information-content equivalent to the tranditional MIO files."
+* insert RequireMetaProfile(CodedContentDocument)
+* insert BundleIdentifiers
+* type 1..1
+* type only code
+* type = #collection (exactly)
+* entry.resource 1..1 // each entry must have a resource
```

Comment:
Just wanted to clarify this based on our conversation this morning:

when should slices be required to have the discriminator field populated (here `resource`)?
why is it that in the profile all slices have `resource` as mandatory/1..1, but the `CodedIndustryAndOccupation` slice specifically does not have require `resource`?

Reply:
In VRDR, it appears that there is a consistent requirement:
 entry.resource 1..1 // each entry must have a resource
 
 The guy who did Messaging was consistent in the other direction (leaving it as 0..1).
 In slicing by Profile I don't think it really makes a difference, since the slice is only present if the content in the slice conforms to the profile, and the only way for it to conform is to exist.   
 
 An entry in a bundle can contain a lot of other stuff:  https://build.fhir.org/bundle.html
 
![image](https://github.com/HL7/vrdr/assets/37216915/31c43975-d526-4157-b79a-57dd6ee7b191)

In this case, we are slicing off of the profile of the resource, so it obviously has to be present.   If we were slicing off of something else, the presence/absence of the resource would be more ambiguous, and requiring the resource might be meaningful.   Good questions.

Reply:
Also, the cardinality of the slice in the bundle should be 1..1.  If there is no content in a bundle, it is a waste of space.
So, another good comment.